test/integration/load_node_test.cpp

```diff
@@ -314,6 +314,8 @@ TEST(NodeTest, IncorrectFlow) {
 TEST(NodeTest, LoadTildeAsNull) {
   Node node = Load("~");
   ASSERT_TRUE(node.IsNull());
+  EXPECT_EQ(node.as<std::string>(), "null");
```

Comment:
Are you sure this is the behavior we want? If `node` is null, shouldn't `node.as<std::string>()` be an error?

Reply:
Never mind, you know what, I see I commented on that bug that this is reasonable. I'll approve.